doc/source/ray-air/examples/tfx_tabular_train_to_serve.ipynb

```diff
@@ -0,0 +1,2059 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "VaFMt6AIhYbK"
+   },
+   "source": [
+    "This notebook is adapted from \"components keras\" [tutorial](https://www.tensorflow.org/tfx/tutorials/tfx/components_keras).\n",
+    "The goal is to predict whether a trip may generate a big tip.\n",
+    "\n",
+    "In this example, we showcase how to convert a tfx pipeline to [Ray AIR](https://docs.ray.io/en/latest/ray-air/getting-started.html), covering\n",
+    "every step from data ingestion to pushing a model to serving.\n",
```

Comment:
```suggestion
    "In this example, we showcase how to convert a TFX pipeline (link to pipeline in code) to [Ray AIR](https://docs.ray.io/en/latest/ray-air/getting-started.html), covering\n",
    "every step from data ingestion to pushing a model to serving.\n",
```

Also btw, it doesn't actually convert a TFX pipeline right? It just demonstrates how you would similarly do it in AIR (but right now the wording of your sentence makes it sound like a step-by-step conversion guide).

Reply:
hmmm, interesting..  could you say more about what would that look like then?
My understanding is that TFX tutorial is to go over all the components (ExampleGen, Trainer, Pusher etc) in a typical TFX pipeline. It's easier for them to tell the component story in an interactive notebook setup. But the difference between that and a TFX pipeline is quite minimal - you just need to connect those components together into a pipeline, by `tfx.dsl.Pipeline()`.
So I would consider our counterpart example does convert a TFX pipeline (each TFX pipeline component is mapped into an AIR component).
Ah or maybe you are thinking more about orchestration part of a TFX pipeline by the wording? The Airflow stuff?